app/Http/Controllers/TopoController.php

```diff
@@ -8,6 +8,8 @@
 use Illuminate\Http\Request;
 use Illuminate\Support\Facades\Auth;
 use Illuminate\Support\Facades\DB;
+use TomLingham\Searchy\Facades\Searchy;
+use Log;
```

Comment:
I don't think it's used.

Reply:
Searchy is used (search topo by name) but yes - log is killed already in the new pull request (https://github.com/lucien-chastan/oblyk/pull/29)